tests/pipelines/sana_video/test_sana_video_i2v.py

```diff
@@ -0,0 +1,235 @@
+# Copyright 2025 The HuggingFace Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+import tempfile
+import unittest
+
+import numpy as np
+import torch
```

Comment:
```suggestion
import torch
from PIL import Image
```
Sorry, this should also imported in conjunction with the change from https://github.com/huggingface/diffusers/pull/12634#discussion_r2517435680